src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Arm/Crc32.PlatformNotSupported.cs

```diff
@@ -0,0 +1,94 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+
+namespace System.Runtime.Intrinsics.Arm
+{
+    /// <summary>
+    /// This class provides access to the ARM Crc32 hardware instructions via intrinsics
+    /// </summary>
+    [Intrinsic]
+    [CLSCompliant(false)]
+    public abstract class Crc32 : ArmBase
+    {
+        internal Crc32() { }
+
+        public static new bool IsSupported { [Intrinsic] get { return false; } }
+
+        [Intrinsic]
+        public new abstract class Arm64 : ArmBase.Arm64
+        {
+            internal Arm64() { }
+
+            public static new bool IsSupported { [Intrinsic] get { return false; } }
+
+            /// <summary>
+            /// uint32_t __crc32d (uint32_t a, uint64_t b)
+            ///   A64: CRC32X Wd, Wn, Xm
+            /// </summary>
+            public static uint Crc32(uint crc, ulong data) { throw new PlatformNotSupportedException(); }
+
+            /// <summary>
+            /// uint32_t __crc32cd (uint32_t a, uint64_t b)
+            ///   A64: CRC32CX Wd, Wn, Xm
+            /// </summary>
+            public static uint Crc32C(uint crc, ulong data) { throw new PlatformNotSupportedException(); }
+        }
+
+        /// <summary>
+        /// uint32_t __crc32b (uint32_t a, uint8_t b)
+        ///   A32: CRC32B Rd, Rn, Rm
+        ///   A64: CRC32B Wd, Wn, Wm
+        /// </summary>
+        public static uint Crc32(uint crc, byte data) { throw new PlatformNotSupportedException(); }
+
+        /// <summary>
+        /// uint32_t __crc32h (uint32_t a, uint16_t b)
+        ///   A32: CRC32H Rd, Rn, Rm
+        ///   A64: CRC32H Wd, Wn, Wm
+        /// </summary>
+        public static uint Crc32(uint crc, ushort data) { throw new PlatformNotSupportedException(); }
+
+        /// <summary>
+        /// uint32_t __crc32w (uint32_t a, uint32_t b)
+        ///   A32: CRC32W Rd, Rn, Rm
+        ///   A64: CRC32W Wd, Wn, Wm
+        /// </summary>
+        public static uint Crc32(uint crc, uint data) { throw new PlatformNotSupportedException(); }
+
+        /// <summary>
+        /// uint32_t __crc32d (uint32_t a, uint64_t b)
+        ///   A64: CRC32X Wd, Wn, Wm
```

Comment:
Here as well.

Reply:
This shouldn't be here at the first place since it's A32 and A64 ISA.
I will remove